src/auto-clicker.ts

```diff
@@ -22,7 +22,7 @@
 
   function clickBonus() {
     const interactiveElement = targetNode.querySelector<HTMLButtonElement>(
-      '.tw-interactive'
+      '.tw-interactive,.tw-button'
```

Comment:
Hi guy, I tried to get this element directly from the browser console in this format but it didn't work for me, maybe you should just put the ".tw-button" in the querySelector because it worked for me in the tests that I did

Reply:
querySelector doesn't work that way, putting a comma for it to get different elements. As this css class has been changed, the ideal is to put only the value of the new class

Reply:
I will leave a comparison of the test I did:

Searching for ".tw-interactive"
<img width="891" alt="Screen Shot 2020-12-18 at 18 40 39" src="https://user-images.githubusercontent.com/8340717/102664218-0bc49d80-4161-11eb-9249-9dbaac558591.png">

Searching for ".tw-button"
<img width="828" alt="Screen Shot 2020-12-18 at 18 41 07" src="https://user-images.githubusercontent.com/8340717/102664233-12ebab80-4161-11eb-955b-8da6c270371a.png">

Reply:
When I search using both it selects only the old class instead of selecting the new one.

<img width="898" alt="Screen Shot 2020-12-18 at 19 03 55" src="https://user-images.githubusercontent.com/8340717/102665585-dd948d00-4163-11eb-9520-54c50a6ea9a9.png">

<img width="641" alt="Screen Shot 2020-12-18 at 19 04 19" src="https://user-images.githubusercontent.com/8340717/102665601-e1281400-4163-11eb-87a5-00b1c2b38623.png">